src/components/Layouts/ReportLayout.less

```diff
@@ -21,16 +21,14 @@ h1 {
   text-transform: capitalize;
 }
 
-.section-layout, .section-markdown {
-  &.not-bordered {
-    border: none !important;
-    padding: 0 !important;
-  }
+.not-bordered {
+  border: none !important;
+  padding: 0 !important;
+}
 
-  &.bordered {
-    border: 1px solid #DADBDB !important;
-    padding: 10px 12px !important;
-  }
+.bordered {
+  border: 1px solid #DADBDB !important;
+  padding: 10px 12px !important;
 }
```

Comment:
so were it used now if we make it available at global scope again?

Reply:
Removed the classes - unused.